torch/distributions/utils.py

```diff
@@ -11,3 +11,27 @@ def expand_n(v, n):
         return torch.Tensor([v]).expand(n, 1)
     else:
         return v.expand(n, *v.size())
+
+
+def broadcast_shape(*shapes):
```

Comment:
Isn't this just `torch._C._infer_size(t1.size(), t2.size())`?

Reply:
I didn't realize there was an already accessible helper utility; will use that instead.